functions.php

```diff
@@ -1,11 +1,28 @@
 <?php
 
+/* the function createDB() allows the program to call the database connection
```

Comment:
this is not how you docblock.
this docblock should look like:
```
*/
* creates a database connection
*/
```

Reply:
done as directed.